src/style.css

```diff
@@ -2,2 +1,83 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.container-logo {
+  display: flex;
+}
+
+.container-logo img {
+  display: block;
+}
+
+.nav {
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  padding: 0;
+  text-align: center;
+  height: 60px;
+  line-height: 60px;
+  margin-left: 20px;
+}
```

Comment:
You are using the tag name `a` to style the navigation links. It's better to style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. In this case, you have the class `.nav__link` which is great.